docker/Dockerfile

```diff
@@ -55,28 +55,28 @@ RUN wget -q https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_
 
 
 # install ExpansionHunterDenovo binary from https://github.com/Illumina/ExpansionHunterDenovo
-ENV VERSION=v0.9.0
+ENV VERSION="v0.9.0"
```

Comment:
basic question, why is it better to have quotes around these?

Reply:
I think they're equivalent. I just switched it for consistency.